foundations/installations/setting_up_git.md

```diff
@@ -182,7 +182,7 @@ ssh-keygen -t ed25519 -C "your@email.com"
 ```
 
 <div class="lesson-note lesson-note--tip" markdown="1">
-The `-C` flag is to write a comment, otherwise the key will be generated with your computer's username. The convention is to use your email as a comment to indicate who generated the public key. For example if your email address is `odin@theodinproject.com`, then you would type `ssh-keygen -t ed25519 -C "odin@theodinproject.com"`.
+The `-C` flag is to write a comment, otherwise the key will be generated with your computer's username. The convention is to use your email as a comment to indicate who generated the public key. For example if your email address is `odin@theodinproject.com`, then you would type `ssh-keygen -t ed25519 -C "odin@theodinproject.com"`. Like before, if you opted to use the private GitHub email address then you should use the private email address GitHub provided you.
```

Comment:
Would simply stating something along the lines of, "You should use the same email that you used to initially setup Git" suffice?

Reply:
@robertung88 checking if you've seen the comment above